packages/gamut-labs/src/ListSection/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactElement, useState } from 'react';
+
+import { PageSection } from '..';
+
+export type ListSectionProps<T> = {
+  title: string;
+  /**
+   * Array of elements.
+   */
+  listItems: T[];
+  /**
+   * Render function that will render each element in the list items.
+   */
+  renderItem: (item: T, index?: number) => ReactElement;
+  /**
+   * Number of items to be initally displayed (note: if the initial display amount matches the number of list items, Show All button will not appear).
```

Comment:
Nit: [TSDoc](https://tsdoc.org/) has a think for this:

```suggestion
   * Number of items to be initally displayed
   *
   * @remarks
   * If the initial display amount matches the number of list items, Show All button will not appear.
```